app.R

```diff
@@ -0,0 +1,1215 @@
+                  #     tags$ul(
+                  #       tags$li("Divide the range of the predictor into intervals with roughly equal numbers of cases."),
+                  #       tags$li("Compute the mean value of the predictor and the empirical logit for each interval."),
+                  #       tags$li("Plot logit versus the mean value of the predictor, with one point for each interval.")
+                  #     ),
+                  #     style = "background-color: #ffb6c1")))),
+                  
+                  #                                        box(width =4,background = "maroon", 
+                  #                                            title = "Process of creating an empirical logit plot for quantitative predictors", "
+                  # 1) Divide the range of the predictor into intervals with roughly equal numbers of cases. 
+                  # 2) Compute the mean value of the predictor and the empirical logit for each interval. 
+                  # 3) Plot logit versus the mean value of the predictor, with one point for each interval."),
+                  #   
+                  
+                  h3("Process of creating an empirical logit plot for quantitative predictors"),
+                  h4("1. Divide the range of the predictor into intervals with roughly equal numbers of cases."),
+                  h4("2. Compute the mean value of the predictor and the empirical logit for each interval."),
+                  h4("3. Plot logit versus the mean value of the predictor, with one point for each interval."),
+                  br(),
+                  #                                        box(width = 6, background = "maroon", title = "How many intervals should we use for the Empirical Logit Plot?",
+                  #                                            "Two intervals give you a sense of the direction and size of the relationship.
+                  # Three intervals give you an indication of departures from linearity. 
+                  # Four or five intervals is better when you have enough cases. "),
+                  #                                        br(),
+                  
+                  
+                  
+                  sidebarLayout(
+                    sidebarPanel(
+                      ####select datasets
+                      selectInput(inputId="datatable", label="Select Dataset:", 
+                                  choices= c('MedGPA', 'Titanic', 'Leukemia'), 
+                                  selected = 'MedGPA'),
+                      
+                      ####variable options for 'MedGPA' dataset
+                      conditionalPanel(
+                        condition = "input.datatable == 'MedGPA'",
+                        selectInput(inputId="MedYvar", label="Select Response Variable Y",
+                                    choices = c("Acceptance"),
+                                    selected = 'Acceptance'),
```

Comment:
Not sure if this selection is needed since there is only one option for the response variable for each of the possible data sets.